Web/jqueryassignment2/qn2/WebContent/Assignment2.js

```diff
@@ -0,0 +1,29 @@
+$(document).ready(function() 
+{
+	$("#empName").change(function() 
+	{
+		loadInfo();
```

Comment:
Good practice is to make methods least dependent on HTML elements. E.g. you could pass the name from the #empName check box to the method 'loadInfo' so that it need not query this tag.